frontend/src/ramp.pug

```diff
@@ -32,6 +32,7 @@
             borderLeftWidth: getWidth(slice) + 'em',\
             right: (getSlicePos(tframe === 'day' ? slice.date : slice.endDate) * 100) + '%' \
           },\
+          /* disallow clickable ramp slices when merging groups that are grouped by authors */\
```

Comment:
Also. add in the rationale that its due to different repository, hence unable to form a url for clickable link